kong/plugins/tcp-log/handler.lua

```diff
@@ -53,6 +74,22 @@ local TcpLogHandler = {
 }
 
 
+function TcpLogHandler:access(conf)
+  if conf.log_body then
+    ngx.ctx.request_body = get_body_data()
+    ngx.ctx.response_body = ""
```

Comment:
this risks name collisions, use the pdk again for plugin storage: https://docs.konghq.com/2.0.x/pdk/kong.ctx/#kongctxplugin

Reply:
Good advice. I'll fix it.

Reply:
Fixed.